src/SFA.DAS.Commitments.Database/StoredProcedures/UpdateDatalockStatusIsExpired.sql

```diff
@@ -0,0 +1,16 @@
+﻿CREATE PROCEDURE [dbo].[UpdateDatalockStatusIsExpired]
+	@ApprenticeshipId bigint ,
+	@PriceEpisodeIdentifier NVARCHAR(255)
+	WITH RECOMPILE
+AS
+	UPDATE [dbo].[DataLockStatus]
+	SET
+		[IsExpired]  = 1,
+		[Expired] = GETDATE()
```

Comment:
We should inject @now into the proc and use that instead of GETDATE(), so that it will work with time machine.